### PR TITLE
[Interop] Rebuild dart image

### PR DESCRIPTION
**Pull request by ejona86**
The last time it was rebuilt, the base image (dart:stable) seems not to have been updated. grpc-dart is now requiring newer dart version and is currently failing: https://github.com/grpc/grpc-dart/issues/705

### CODE REVIEW

**Review by veblush — APPROVED** [review]

File: tools/bazelify_tests/dockerimage_current_versions.bzl [hunk 1]
```diff
@@ -72,7 +72,7 @@ DOCKERIMAGE_CURRENT_VERSIONS = {
     "tools/dockerfile/grpc_artifact_python_musllinux_1_1_x86.current_version": "docker://us-docker.pkg.dev/grpc-testing/testing-images-public/grpc_artifact_python_musllinux_1_1_x86@sha256:c288f83435186ee675d004ee52c93195a51201bf2b5fe92581584d977a2499a3",
     "tools/dockerfile/interoptest/grpc_interop_aspnetcore.current_version": "docker://us-docker.pkg.dev/grpc-testing/testing-images-public/grpc_interop_aspnetcore@sha256:8e2e732e78724a8382c340dca72e7653c5f82c251a3110fa2874cc00ba538878",
     "tools/dockerfile/interoptest/grpc_interop_cxx.current_version": "docker://us-docker.pkg.dev/grpc-testing/testing-images-public/grpc_interop_cxx@sha256:e5a474d33773d52ec6a8abbe2d61ee0c2a9c2b5f48793a5ea3b82c4445becf3f",
-    "tools/dockerfile/interoptest/grpc_interop_dart.current_version": "docker://us-docker.pkg.dev/grpc-testing/testing-images-public/grpc_interop_dart@sha256:2051ef1cf81c463a65853f6058135f48e52a7572cc2bad57182256b77ef61021",
+    "tools/dockerfile/interoptest/grpc_interop_dart.current_version": "docker://us-docker.pkg.dev/grpc-testing/testing-images-public/grpc_interop_dart@sha256:c44c44706e28c328489ab9cbbf1f2940855c4c341639fbda76b2fc1d561e3383",
```
Comment (yashykt) [inline tools/bazelify_tests/dockerimage_current_versions.bzl:75]: is that the right sha digest?

Comment (yashykt) [reply tools/bazelify_tests/dockerimage_current_versions.bzl:75]: I'm seeing 4915a280788126dad029858eb384dbbef2dc18cadccb434df6450dfd7a4929f2

Comment (ejona86) [reply tools/bazelify_tests/dockerimage_current_versions.bzl:75]: You could have gotten a slightly different dart:stable.

Comment (yashykt) [reply tools/bazelify_tests/dockerimage_current_versions.bzl:75]: It's the digest from the artifact registry website for `us-docker.pkg.dev/grpc-testing/testing-images-public/grpc_interop_dart:c795310ab447f964b848b656e083196e0ece2dfc`

**Comment by ejona86** [comment]
Copybara didn't actually commit the full changes from this PR. Created https://github.com/grpc/grpc/pull/36562 to fix it.

**Pull request closed without merging**